x/emissions/keeper/genesis.go

```diff
@@ -1345,6 +1367,25 @@ func (k *Keeper) ExportGenesis(ctx context.Context) (*types.GenesisState, error)
 		return nil, errors.Wrap(err, "failed to get previous percentage reward to staked reputers")
 	}
 
+	// openWorkerWindows
+	openWorkerWindows := make([]*types.BlockHeightAndTopicIds, 0)
```

Comment:
Do we really need to create a specific structure just for the import/export? Is it the only way?

Reply:
I don't really understand the import export, but based on my partial understanding yes, this is what we have to do